src/templates/LabOverviewTemplate.js

```diff
@@ -47,21 +47,29 @@ const LabOverviewTemplate = ({ data, location }) => {
               <GuideListing.List className={labOverviewStyles.labGuideList}>
                 {guides?.nodes
                   .sort(sortProcedures)
-                  .map(({ fields, frontmatter }, index) => (
-                    <GuideTile
-                      className={labOverviewStyles.labGuideCard}
-                      to={fields.slug}
-                      key={index}
-                      duration={frontmatter.duration}
-                      title={`${frontmatter.procIdx}. ${
-                        frontmatter.tileShorthand?.title || frontmatter.title
-                      }`}
-                      description={
-                        frontmatter.tileShorthand?.description ||
-                        frontmatter.description
-                      }
-                    />
-                  ))}
+                  .map(({ fields, frontmatter }, index) => {
```

Comment:
Attached is the screen shot for this change so it's visible, this is OK for me, so I approve this.
<img width="1109" alt="Screen Shot 2022-02-04 at 9 47 45 AM" src="https://user-images.githubusercontent.com/1395158/152585825-0ecb3462-337b-426d-8794-11162fd83217.png">